src/Request/ConsoleRequest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Bugsnag\Request;
+
+class ConsoleRequest implements RequestInterface
+{
+    /**
+     * The unformated console command.
+     *
+     * @var string[]
+     */
+    protected $command;
+
+    /**
+     * Create a new console request instance.
+     *
+     * @param string[] $command an array of the console command input
+     *
+     * @return void
+     */
+    public function __construct(array $command)
+    {
+        $this->command = $command;
+    }
+
+    /**
+     * Are we currently processing a request?
+     *
+     * @return bool
+     */
+    public function isRequest()
+    {
+        return false;
+    }
+
+    /**
+     * Get the session data.
+     *
+     * @return array
+     */
+    public function getSession()
+    {
+        return [];
+    }
+
+    /**
+     * Get the cookies.
+     *
+     * @return array
+     */
+    public function getCookies()
+    {
+        return [];
+    }
+
+    /**
+     * Get the request formatted as meta data.
+     *
+     * @return array
+     */
+    public function getMetaData()
+    {
+        if (is_array($this->command)) {
+            $commandString = implode(' ', $this->command);
+            $primaryCommand = $this->command[0];
+            $arguments = [];
+            $options = [];
+            foreach (array_slice($this->command, 1) as $arg) {
+                $arg[0] == '-' ? $options[] = $arg : $arguments[] = $arg;
+            }
+            $data = [
+                'Input' => $commandString,
+                'Command' => $primaryCommand,
+                'Arguments' => $arguments,
+                'Options' => $options,
+            ];
+        } else {
+            $data = $this->command;
+        }
+
+        return ['console' => $data];
+    }
+
+    /**
+     * Get the request context.
+     *
+     * @return string|null
+     */
+    public function getContext()
+    {
+        return implode(' ', array_slice($this->command, 0, 2));
```

Comment:
Just realized that php doesn't include the executable in the arguments list. Maybe we should just take the first part of the array then, or implode the whole thing. Ping @kattrali - what do you think is the most sensible option here?

Reply:
I think first few args still works well enough. The cases I can try/think of:
* `php [filename]` => `[filename]`
* `php [filename] foo bar` => `[filename] foo`
* `php [filename] --foo bar --baz red` => `[filename] --foo` <-- this one is a lil odd
* `php artisan migrate` => `artisan migrate`

I could see the value in imploding the whole thing, but also don't want context to get arbitrarily long. We could max four args for now and revisit with feedback.

Reply:
> but also don't want context to get arbitrarily long

Yeh, totally. We need to find the balance where it is useful but not unbounded.